apps/financial-aid/web-veita/src/components/UsersTableBody/UsersTableBody.tsx

```diff
@@ -67,11 +94,27 @@ const UsersTableBody = ({ user, index }: PageProps) => {
             })}
           >
             {user.active ? (
-              <Button variant="text" colorScheme="destructive">
+              <Button
+                onClick={(event) => {
+                  event.stopPropagation()
+                  changeUserActivity(false)
+                }}
+                variant="text"
+                loading={loading}
+                colorScheme="destructive"
+              >
                 Óvirkja
               </Button>
             ) : (
-              <Button variant="text" colorScheme="light">
+              <Button
+                onClick={(event) => {
+                  event.stopPropagation()
+                  changeUserActivity(true)
+                }}
+                variant="text"
+                loading={loading}
+                colorScheme="light"
+              >
```

Comment:
Instead of having two buttons, maybe have one: f.x.  🤷‍♀️ 
     <Button
                onClick={(event) => {
                  event.stopPropagation()
                  changeUserActivity(user.active)
                }}
                variant="text"
                loading={loading}
                colorScheme={user.active ? "destructive": "light"}
              >
{user.active ? "Óvirkja": "Virkja"}
              </Button>